### DISCUSSION TITLE
Visualization error or bad table defintions

### DISCUSSION THREAD

**Discussion by GiuseppeDiGuglielmo** [dbeaver/dbeaver#21822] [General]
The following db has some weird text on the dependencies , e.g. (1). 

Is it because of the circular dependency (2) (3)? Or is it some issue with dbeaver?

My goal was:
```
model      1-to-n   model_qor
model      1-to-n   layer
layer      1-to-n   layer_qor
model_qor  1-to-n   layer_qor
```

![2023-11-10_17-05](https://github.com/dbeaver/dbeaver/assets/3743785/792dd7ab-3983-4644-85e5-c472df60fe36)

**Comment by LonwoLonwo** [comment]
Could you please provide more explanations?
